Dockerfile.dapper

```diff
@@ -19,7 +19,7 @@ RUN apt-get update -qq && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 # install yq
-RUN GO111MODULE=on go get github.com/mikefarah/yq/v4@v4.6.0
+RUN GO111MODULE=on go install github.com/mikefarah/yq/v4@v4.6.0
```

Comment:
From 1.18, `go install` is the favoured command to install executable.

Ref: https://go.dev/doc/go1.18#go-get